packages/side-nav/src/vaadin-side-nav-item.js

```diff
@@ -190,7 +216,7 @@ class SideNavItem extends SideNavChildrenMixin(DisabledMixin(ElementMixin(Themab
   updated(props) {
     super.updated(props);
 
-    if (props.has('path') || props.has('pathAliases')) {
+    if (props.has('path') || props.has('pathAliases') || props.has('exact')) {
```

Comment:
Should also use `matchExact`. Also, apparently this isn't currently tested as the name here is wrong.

Reply:
Fixed and added a test